pemFioi/taskStyles-mobileFirst.css

```diff
@@ -442,3 +443,23 @@ img {
 .pythonIntroFull code.pflForbidden {
   cursor: not-allowed;
 }
+#editExerciseTitle {
```

Comment:
It should be in `quickAlgo-mobileFirst.css`, as it's CSS related to quickAlgo's interface.